editor/configs/locationCrashesTable.ts

```diff
@@ -73,4 +73,39 @@ export const locationCrashesQueryConfig: QueryConfig = {
     }),
   },
   filterCards: locationCrashesFiltercards,
+  mapConfig: {
```

Comment:
I was going to ask if we could default this map to the aerial imagery basemap instead of streets, but it looks like that will require some refactor of the TableMap component that feels beyond scope. Right?